django/contrib/admin/templates/admin/delete_confirmation.html

```diff
@@ -20,21 +20,21 @@
 
 {% block content %}
 {% if perms_lacking %}
-    <p>{% blocktrans with escaped_object=object %}Deleting the {{ object_name }} '{{ escaped_object }}' would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:{% endblocktrans %}</p>
+    <p>{% blocktrans with escaped_object=object force_escape %}Deleting the {{ object_name }} '{{ escaped_object }}' would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:{% endblocktrans %}</p>
```

Comment:
Not sure. I feel like we don't have an elegant solution yet. If you say we must always use `{% force_escape %}` in HTML, then how do we handle the case Claude mentioned: `{% blocktrans %}This is <b>important</b>{% endblocktrans %}` We don't want the `<b>` tags escaped but we want other characters escaped.